rest/config_test.go

```diff
@@ -862,3 +863,41 @@ func TestValidateServerContext(t *testing.T) {
 	assert.Equal(t, testDataBucket, SharedBucketError.GetSharedBucket().bucketName)
 	assert.Subset(t, []string{"db1", "db2"}, SharedBucketError.GetSharedBucket().dbNames)
 }
+
+func TestParseCommandLineWihIllegalOptionBucket(t *testing.T) {
+	var (
+		adminInterface     = "127.0.0.1:4985"
+		bucket             = "sync_gateway"
+		cacertpath         = "/etc/ssl/certs/ca.cert"
+		certpath           = "/etc/ssl/certs/client.pem"
+		configServer       = "http://127.0.0.1:4981/conf"
+		dbname             = "beer_sample"
+		defaultLogFilePath = "/var/log/sync_gateway"
+		deploymentID       = "DEPID100"
+		interfaceAddress   = "4984"
+		keypath            = "/etc/ssl/certs/key.pem"
+		logKeys            = "Admin,Access,Auth,Bucket"
+		logFilePath        = "/var/log/sync_gateway"
+		pool               = "liverpool"
+	)
+	args := []string{
+		"sync_gateway",
+		"--adminInterface", adminInterface,
+		"--bucket", bucket, // Bucket option has been removed
+		"--cacertpath", cacertpath,
+		"--certpath", certpath,
+		"--configServer", configServer,
+		"--dbname", dbname,
+		"--defaultLogFilePath", defaultLogFilePath,
+		"--deploymentID", deploymentID,
+		"--interface", interfaceAddress,
+		"--keypath", keypath,
+		"--log", logKeys,
+		"--logFilePath", logFilePath,
+		"--pool", pool,
+		"--pretty"}
+
+	config, err := ParseCommandLine(args, flag.ContinueOnError)
+	assert.Error(t, err, "Parsing commandline arguments without any config file")
+	assert.Empty(t, config, "Couldn't parse commandline arguments")
```

Comment:
Given that you're not asserting anything other than error, seems like you could remove most of the args for this test.

Reply:
Makes sense; Fixed in latest revision.